proto/sf/substreams/rpc/v2/service.proto

```diff
@@ -130,6 +130,37 @@ message OutputDebugInfo {
 
 message ModulesProgress {
   repeated ModuleProgress modules = 1;
+  repeated Job running_jobs = 2;
+  repeated ModuleStats modules_stats = 3;
+}
+
+message Job {
+    uint64 stage = 1;
+    repeated string modules = 2;
+    uint64 start_block = 3;
+    uint64 stop_block = 4;
+    uint64 processed_blocks = 5;
+    uint64 duration_secs = 6;
+}
+
+message ModuleStats {
+    string name = 1;
+
+    // total_processed_blocks includes duplicate for each stage going over that module
+    uint64 total_processed_blocks = 2;
+    uint64 highest_processed_block = 3;
+    uint64 total_processing_time_secs = 4;
+    uint64 total_external_calls_time_secs = 5;
+    uint64 total_external_calls_count = 6;
+    uint64 total_store_operations_time_secs = 7;
+    uint64 total_store_reads_count = 8;
+    uint64 total_errors_count = 9;
```

Comment:
What is this, seems weird as a progress, what do you see as being tracked here, retry of a job?

Reply:
yes, an error on a module on a tier2, summed up here...
Allows the CLI to report back to the user if a specific module caused errors... Maybe it is not relevant, but since there are retries, it could be interesting information when debugging.

Reply:
There's a convention for those `_count` suffixes, it's that it's always singular before the `_count`.. so that would be, for both: `total_store_read_count` and `total_error_count` .. let's unify all the rest this way too.